api-report/map.api.md

```diff
@@ -37,7 +38,7 @@ export class DirectoryFactory {
 }
 
 // @public
-export interface IDirectory extends Map<string, any>, IEventProvider<IDirectoryEvents> {
+export interface IDirectory extends Map<string, any>, IEventProvider<IDirectoryEvents>, IDisposable {
```

Comment:
this is a breaking change. it needs to be Partial<IDisposable> to be non-breaking